CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.6.2
 
+### Added
+
+- Ability to disable X11 or Wayland backends
```

Comment:
I feel like we should add that this is behind compile-time feature flags. Not runtime options or anything like that.

```suggestion
- Features `x11` and `wayland` for disabling the linux backends
```